### PR TITLE
Show folders' files when selecting a folder

### PR DESCRIPTION
**Pull request by alex-deaconu**
[stage-4]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: web/partials/storage/files-list.html [hunk 1]
```diff
@@ -2,93 +2,94 @@
...
+                </span>
               </span>
+            </a>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr class="clickable-row"
+        ng-click="fileClick(file);"
+        ng-class="{'active': file.isChecked, 'blocked-file': file.isThrottled, 'disabled-row': !storageFactory.fileIsFolder(file) && storageFactory.isSingleFolderSelector(), 'back-btn': file.currentFolder, 'no-select-row': file.currentFolder || storageFactory.fileIsTrash(file) || (!storageFactory.fileIsFolder(file) && storageFactory.isSingleFolderSelector()) || (storageFactory.fileIsFolder(file) && !storageFactory.isSingleFolderSelector()) }"
```
Comment (ezequielc) [inline web/partials/storage/files-list.html]: @alex-deaconu I think we need `(storageFactory.fileIsFolder(file) && !storageFactory.storageFull)` here. In this version, I can no longer select Folders to download in Apps Storage. Thanks

Comment (alex-deaconu) [inline web/partials/storage/files-list.html]: @ezequielc good catch. Will take a look. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc I've actually split out that particular functionality from the partial into the controller. This allows for unit testing of the different scenarios. Please take a look. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu Nice improvement! :+1: Thanks!

**Pull request merged**